.github/workflows/test.yml

```diff
@@ -5,9 +5,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
```

Comment:
@semanticart I also usually configure the `node-version-file` with a `.tool-versions` file, but wasn't sure what version of node you guys would want to pin to, so left out for now. Let me know thoughts.

Reply:
That's fine by me. I've been developing with nodejs 18.4.0 locally. Anything after the inclusion of `fetch` in node core is 👍

Reply:
OK, I will add this now then